daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -993,22 +912,22 @@ final class ChoiceRuntimeData(
    * hook these objects into a parent-child tree without
```

Comment:
Fix comment. Not really about parent-child tree, but the fact that these provide access to objects which direclty or indirectly refer to this object.

Reply:
I think so. One reason for the cycles was, in the past parent/child doubly-linked structure. We don't have that now so that reason isn't there any more, which is why the comment is no longer relevant.